hoot-core-test/src/test/resources/cmd/slow/MergePoisTest.sh.stdout

```diff
@@ -1,22 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <osm version="0.6" generator="hootenanny" srs="GEOGCS[&quot;WGS 84&quot;,&#10;    DATUM[&quot;WGS_1984&quot;,&#10;        SPHEROID[&quot;WGS 84&quot;,6378137,298.257223563,&#10;            AUTHORITY[&quot;EPSG&quot;,&quot;7030&quot;]],&#10;        TOWGS84[0,0,0,0,0,0,0],&#10;        AUTHORITY[&quot;EPSG&quot;,&quot;6326&quot;]],&#10;    PRIMEM[&quot;Greenwich&quot;,0,&#10;        AUTHORITY[&quot;EPSG&quot;,&quot;8901&quot;]],&#10;    UNIT[&quot;degree&quot;,0.0174532925199433,&#10;        AUTHORITY[&quot;EPSG&quot;,&quot;9108&quot;]],&#10;    AUTHORITY[&quot;EPSG&quot;,&quot;4326&quot;]]">
-    <node visible="true" id="-1" timestamp="1970-01-01T00:00:00Z" version="1" lat="48.0479399000000029" lon="11.7012813999999992">
-        <tag k="hoot:status" v="1"/>
+    <node visible="true" id="-2" timestamp="1970-01-01T00:00:00Z" version="1" lat="48.0485400000000027" lon="11.7019400000000005">
+        <tag k="hoot:status" v="2"/>
         <tag k="name" v="POST, JÄGER-VON-FALL-STRASSE"/>
         <tag k="amenity" v="post_office"/>
-        <tag k="error:circular" v="15"/>
+        <tag k="error:circular" v="150"/>
     </node>
```

Comment:
@mingjiesu This relates back to the ordering ticket that Brian mentioned. I'm afraid that merging this into develop may make that problem worse. I don't have redmine in front of me now, but we should chat about this before this is merged.